### PR TITLE
Reboot AKO when there is error in retrieving out Openshift clientset or informer in Openshift environment

### PR DESCRIPTION
**Pull request by akshayhavile**
…

### CODE REVIEW

**Comment by Dhivyaaj** [comment]
No JIRA Ids found for the PR. Jira id is mandatory to update fix version in jira.                                            Please update respective Jira id in PR title or commit message                                            if the PR is intented for default branches of repo.                                            For mandatory exemptions, comment trigger phrase 'skip jira-id-check' in PR.                                            For manual trigger, comment trigger phrase 'run jira-id-check' in PR.

**Comment by akshayhavile** [comment]
skip jira-id-check

**Comment by akshayhavile** [comment]
build ako

File: cmd/ako-main/main.go [hunk 1]
```diff
@@ -166,7 +166,11 @@ func InitializeAKC() {
 	if err != nil {
 		utils.AviLog.Warnf("Error in creating openshift clientset")
 	}
-
+	if lib.GetCNIPlugin() == lib.OPENSHIFT_CNI || lib.GetCNIPlugin() == lib.OVN_KUBERNETES_CNI {
```
Comment (swathinsankaran) [inline cmd/ako-main/main.go:169]: Shall we club these two statements since both are related?

**Comment by Dhivyaaj** [comment]
No JIRA Ids found for the PR. Jira id is mandatory to update fix version in jira.                                            Please update respective Jira id in PR title or commit message                                            if the PR is intented for default branches of repo.                                            For mandatory exemptions, comment trigger phrase 'skip jira-id-check' in PR.                                            For manual trigger, comment trigger phrase 'run jira-id-check' in PR.

File: cmd/ako-main/main.go (hunk 1 above)
Comment (akshayhavile) [reply cmd/ako-main/main.go]: addressed

**Comment by akshayhavile** [comment]
skip jira-id-check

**Comment by akshayhavile** [comment]
build ako

**Review by arihantg — APPROVED** [review]

**Review by DixitAakash — APPROVED** [review]
LGTM

**Review by swathinsankaran — APPROVED** [review]

**Pull request merged**